Cargo.toml

```diff
@@ -45,7 +45,7 @@ resolver = "2"
 [workspace.package]
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2021"
-repository = "https://github.com/paritytech/frontier/"
+repository = "https://github.com/CESSProject/frontier/"
```

Comment:
This and below need to switch to upstream to be mergeable. Might be a good time to update this to https://github.com/polkadot-evm/frontier too.